src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public static final int MAGIC_NUMBER = 47;
+    private final Random random = new Random(MAGIC_NUMBER);
```

Comment:
Using a fixed seed for the `Random` instance may lead to predictable results which is not usually desirable when generating random colors. Consider using a `new Random()` without a fixed seed unless there's a specific requirement for reproducibility.